testing/src/cypress/e2e/02-institution/02-dashboard-institution-page.cy.ts

```diff
@@ -1,18 +1,17 @@
 import DashboardInstitutionObject from "../../page-objects/Institution-objects/DashboardInstitutionObject";
-import InstitutionCustomCommand from "../../custom-command/institution/InstitutionCustomCommand";
 import ManageInstitutionObject from "../../page-objects/Institution-objects/ManageInstitutionObject";
-import InstitutionHelperActions from "./common-helper-functions.cy";
+import InstitutionHelperActions from "../../custom-command/institution/common-helper-functions.cy";
 
 const dashboardObject = new DashboardInstitutionObject();
 const manageInstitutionObject = new ManageInstitutionObject();
 const institutionHelperActions = new InstitutionHelperActions();
 
-describe("Dashboard Page", () => {
+describe("[Institution Dashboard] - fields and titles ", () => {
   beforeEach(() => {
     institutionHelperActions.loginIntoInstitutionSingleLocation();
   });
 
-  it("Verify that user is redirected to dashboard page", () => {
+  it("[Institution Dashboard] - Verify that user is redirected to dashboard page", () => {
```

Comment:
I think you can remove [Institution Dashboard] in all the individual test components since you already have it in the main test suite (describe) .

Reply:
Yeah true. updating it